packages/material-ui/test/integration/Select.test.js

```diff
@@ -130,8 +130,6 @@ describe('<Select> integration', () => {
             <MenuItem value={10}>Ten</MenuItem>
           </Select>
         </FormControl>,
-        // StrictModeViolation: Requires fake timers + act
-        { strict: false },
```

Comment:
Seems to have been fixed somewhere in between now and https://github.com/mui-org/material-ui/pull/21817